src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -132,5 +133,9 @@ public function process(ContainerBuilder $container)
             $container->getDefinition('twig.runtime.serializer')->addTag('twig.runtime');
             $container->getDefinition('twig.extension.serializer')->addTag('twig.extension');
         }
+
+        if (!$container->has(ImportMapManager::class)) {
```

Comment:
class_exists instead + move this with the other "removeDefinition" above?

Reply:
but I think I miss getting why this needs to be in a compiler pass at all :) 
isn't the conditional check in TwigExtension enough?

Reply:
According to my testing, no it's not enough (I don't know why).